config/config.go

```diff
@@ -36,22 +38,24 @@ func Get() (*Config, error) {
 	}
 
 	cfg = &Config{
-		ZebedeeURL:             "http://localhost:8082",
-		ZebedeeTimeout:         2 * time.Minute,
-		ElasticSearchURL:       "http://localhost:11200",
-		SignESRequests:         false,
-		AwsRegion:              "eu-west-2",
-		AwsSecSkipVerify:       false,
-		DatasetAPIURL:          "http://localhost:22000",
-		ServiceAuthToken:       "",
-		PaginationLimit:        500,
-		MaxDocumentExtractions: 100,
-		MaxDocumentTransforms:  20,
-		MaxDatasetExtractions:  20,
-		MaxDatasetTransforms:   10,
-		TrackerInterval:        5 * time.Second,
-		TopicAPIURL:            "http://localhost:25300",
-		TopicTaggingEnabled:    false,
+		ZebedeeURL:              "http://localhost:8082",
+		ZebedeeTimeout:          2 * time.Minute,
+		ElasticSearchURL:        "http://localhost:11200",
+		SignESRequests:          false,
+		AwsRegion:               "eu-west-2",
+		AwsSecSkipVerify:        false,
+		DatasetAPIURL:           "http://localhost:22000",
+		ServiceAuthToken:        "",
+		PaginationLimit:         500,
+		MaxDocumentExtractions:  100,
+		MaxDocumentTransforms:   20,
+		MaxDatasetExtractions:   20,
+		MaxDatasetTransforms:    10,
+		TrackerInterval:         5 * time.Second,
+		TopicAPIURL:             "http://localhost:25300",
+		TopicTaggingEnabled:     false,
+		EnableDatasetAPIReindex: true,
+		EnableZebedeeReindex:    true,
```

Comment:
Just wondering if these should default to false. My thinking is that as a default, running `make debug` locally would run with fewer dependencies as a default and then you'd only have to run the dependencies you wanted to test initially by enabling just the reindex types you want. We can then add secrets to turn on both of them in the envs. What do you think?

Reply:
yes I agree 👍

Reply:
done